### PR TITLE
moderner-cv:0.1.0

### PR DESCRIPTION
**Pull request by pavelzw**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This is a typst adaptation of LaTeX's [moderncv](https://github.com/moderncv/moderncv), a modern curriculum vitae class.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

---

There is already modern-cv which is why I chose moderner-cv. Is moderner-cv fine? I would like to have something close to LaTeX's package `moderncv` as the name.

### CODE REVIEW

File: packages/preview/moderner-cv/0.1.0/lib.typ [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+#let moderner-cv(
+  name: [],
+  subtitle: [CV],
+  social: (:),
+  color: moderncv-blue,
+  lang: "en",
+  font: ("New Computer Modern"),
+  show_footer: true,
```
Comment (elegaanz) [inline packages/preview/moderner-cv/0.1.0/lib.typ:70]: This argument should be called `show-footer` if you want to be consistent with the rest of the package.

**Review by elegaanz — COMMENTED** [review]
A small naming suggestion, other than that it looks good to me. And the name is fine :)

File: packages/preview/moderner-cv/0.1.0/lib.typ (hunk 1 above)
Comment (pavelzw) [reply packages/preview/moderner-cv/0.1.0/lib.typ]: Thanks, done!

**Comment by elegaanz** [comment]
Thanks!

**Pull request merged**